pulp_container/app/redirects.py

```diff
@@ -60,7 +68,8 @@ class S3StorageRedirects(CommonRedirects):
 
     def issue_tag_redirect(self, tag):
         """
-        Issue a redirect or perform a schema conversion if an accepted media type requires it.
+        Issue a redirect if an accepted media type requires it or return not found if manifest
+        version is not supported.
```

Comment:
your changes are not really related but i noticed and think that we should swap the if/elif branches